docs/usage.rst

```diff
@@ -434,14 +434,18 @@ However, this will never output colorful logs:
     nox --nocolor
 
 
-.. _opt-report:
```

Comment:
If I understand correctly, this link was placed at the wrong location and should have been at the "Outputting a machine-readable report" section, so I've moved it there.